specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2016-10-01/examples/createSecret.json

```diff
@@ -0,0 +1,33 @@
+{    
+    "parameters":{
+        "resourceGroupName":"sample-group",
+        "vaultName":"sample-vault",
+        "secretName":"secret-name",
+        "subscriptionId":"00000000-0000-0000-0000-000000000000",
+        "api-version":"2016-10-01",
+        "parameters":{
+            "properties":{
+                "value": "secret-value"
+            }
+        }
+     },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sample-group/providers/Microsoft.KeyVault/vaults/sample-vault/secrets/secret-name",
+                "name": "secret-name",
+                "type": "Microsoft.KeyVault/vaults/secrets",
+                "location": "westus",
+                "properties": {
+                    "attributes": {
```

Comment:
Missing required property "value".

Reply:
value is not actually required in this model. I've update the model to reflect that.